x-pack/plugins/cloud_security_posture/common/runtime_mappings/get_identifier_runtime_mapping.ts

```diff
@@ -28,7 +29,11 @@ export const getIdentifierRuntimeMapping = (): MappingRuntimeFields => ({
           def policy_template_type = doc["rule.benchmark.posture_type"].value;
 
           if (policy_template_type == "cspm") {
-            emit(doc["cloud.account.id"].value);
+            if (cloudAccountIdAvailable && doc["cloud.account.id"].value != "") {
```

Comment:
Please add a comment describing we check for empty value for backward compatibility of <8.13